proposed/2021/MachineReadableOutput.md

```diff
@@ -0,0 +1,807 @@
+
+The following sources were used:
+   https://api.nuget.org/v3/index.json
+   https://apidev.nugettest.org/v3-index/index.json
+
+No packages were found for the project `MyProjectA` given the specified frameworks.
+Project `MyProjectB` has the following updates to its packages
+   [net5.0]:
+   Top-level Package      Requested             Resolved              Latest
+   > NuGet.Commands       4.8.0-preview3.5278   4.8.0-preview3.5278   6.0.0
+   > Text2Xml.Lib         1.1.2                 1.1.2                 1.1.4
+
+   Transitive Package                                                                   Resolved              Latest
+   > Microsoft.CSharp                                                                   4.0.1                 4.7.0
+   > Microsoft.NETCore.Platforms                                                        1.1.0                 6.0.1
+   > Microsoft.NETCore.Targets                                                          1.1.0                 5.0.0
+
+#### `> dotnet list package --include-transitive --outdated --framework net5.0 --format json`
+
+```json
+{
+  "version": 1,
+  "parameters": "-include-transitive --outdated --framework net5.0",
+   "sources": [
+    "https://api.nuget.org/v3/index.json",
+    "https://apidev.nugettest.org/v3-index/index.json"
+  ],
+  "projects": [
+    {
+      "path": "src/lib/MyProjectA.csproj",
+      "frameworks": [
+      ]
+    },
+    {
+      "path": "src/lib/MyProjectB.csproj",
+      "frameworks": [
+        {
+          "framework": "net5.0",
+          "topLevelPackages": [ 
+              {
+                "id": "NuGet.Commands",
+                "requestedVersion": "4.8.0-preview3.5278",
+                "resolvedVersion": "4.8.0-preview3.5278",
+                "latestVersion": "6.0.0",
+              },
+              {
+                "id": "Text2Xml.Lib",
+                "requestedVersion": "1.1.2",
+                "resolvedVersion": "1.1.2",
+                "latestVersion": "1.1.4",
+              }
+            ],
+            "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1",
+                "latestVersion": "4.7.0",
+              }
+      ...
+            ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### `> dotnet list package --include-transitive --deprecated --framework net5.0`
+
+The following sources were used:
+   https://api.nuget.org/v3/index.json
+   https://apidev.nugettest.org/v3-index/index.json
+
+No packages were found for the project `MyProjectA` given the specified frameworks.
+Project `MyProjectB` has the following deprecated packages
+   [net5.0]:
+   Top-level Package      Requested   Resolved   Reason(s)   Alternative
+   > NuGet.Core           2.13.0      2.13.0     Legacy
+
+   Transitive Package          Resolved              Reason(s)   Alternative
+   > NuGet.Packaging.Core      4.8.0-preview3.5278   Legacy      NuGet.Packaging >= 0.0.0
```

Comment:
Formatting: also need triple-backticks here for consistent formatting

Reply:
Thank you, just pushed fix commit.